crates/storage/db/src/tables/codecs/mod.rs

```diff
@@ -4,6 +4,6 @@ mod compact;
 pub use compact::CompactU256;
 
 pub mod fuzz;
+
 mod postcard;
-#[cfg(not(feature = "bench-postcard"))]
 mod scale;
```

Comment:
And removed this `cfg` as scale implements some of our `Compress/Decompress` traits